src/Feature/Layout/Feature_Layout.re

```diff
@@ -1,13 +1,20 @@
 // MODEL
 
-type model = Layout.t(int);
+type model = {
+  tree: Layout.t(int),
+  uncommittedTree: option(Layout.t(int)),
```

Comment:
Cool idea to have a 'working' tree, and then committing it once the resize has completed